docs/source/getting_started/overview.rst

```diff
@@ -48,6 +47,22 @@ JupyterLab is served from the same `server
 `notebook document format <http://nbformat.readthedocs.io/en/latest/>`__ as the
 classic Jupyter Notebook.
 
+.. _stability:
+
+Stability of JupyterLab
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The current beta release of JupyterLab is stable for daily use.  
```

Comment:
By splitting the stability from the beta section- we would need to make more caveats in this section regarding API stability.